src/core/function_elt.tf

```diff
@@ -74,13 +74,31 @@ module "function_elt" {
     TARGETKAFKA_idempotent          = "true"
     TARGETKAFKA_transactionalId     = "IO_ELT"
     TARGETKAFKA_topic               = "io-cosmosdb-services"
-    ERROR_STORAGE_ACCOUNT           = module.storage_account_elt.name
-    ERROR_STORAGE_KEY               = module.storage_account_elt.primary_access_key
-    ERROR_STORAGE_TABLE             = azurerm_storage_table.fnelterrors.name
-    COMMAND_STORAGE                 = module.storage_account_elt.primary_connection_string
-    COMMAND_STORAGE_TABLE           = azurerm_storage_table.fneltcommands.name
-    IMPORT_TOPIC_NAME               = "import-command"
-    IMPORT_TOPIC_CONNECTION_STRING  = module.event_hub.keys["import-command.io-fn-elt"].primary_connection_string
+
+    MESSAGES_TOPIC_NAME              = "pdnd-io-cosmosdb-messages"
+    MESSAGES_TOPIC_CONNECTION_STRING = module.event_hub.keys["io-cosmosdb-services.io-fn-elt"].primary_connection_string
```

Comment:
Point to the wrong event hub topic. Should point to _pdnd-io-cosmosdb-messages.io-fn-elt_